client/src/pages/TaskHistoryPage/TaskHistoryPage.js

```diff
@@ -1,23 +1,76 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import styles from "./TaskHistoryPage.module.css";
 import TaskListView from "../../components/TaskListView/TaskListView.js";
+import {
+  buildDateRangeRoute,
+  getWeekRange,
+  getMonthRange,
+} from "../../utils.js";
+import { ErrorMessage } from "../../components/ErrorMessage/ErrorMessage.js";
+import { instance } from "../../axios.js";
+import { TaskFilterSelector } from "../../components/TaskFilterSelector/TaskFilterSelector.js";
 
 const TaskHistoryPage = () => {
-  const sampleTask = {
-    title: "gdsc meeting",
-    location: "dh2020",
-    startTime: "05:00pm",
-    endTime: "07:00pm",
-    isOngoing: false,
+  const filterSet = ["Day", "Week", "Month", "All"];
+  const [filter, setFilter] = useState(filterSet[0]);
```

Comment:
I would name this to `currentFilter` as well

Reply:
Renamed `filter` to `currentFilter`.